### PR TITLE
Add conference service listen method to Android

### PR DESCRIPTION
**Pull request by kdrob-dolby**
- Added listen method on android
- Removed unused parameters (preferSendMono, preferRecvMono), added `videoForwardingStrategy` to `join` and `listen` methods.

### CODE REVIEW

File: lib/src/sdk_api/models/conference.dart [hunk 1]
```diff
@@ -210,11 +210,8 @@ class ConferenceJoinOptions {
   /// Allows joining a conference as a special participant called Mixer. For more information, see the [Recording Conferences](https://docs.dolby.io/communications-apis/docs/guides-recording-conferences) article.
   ConferenceMixingOptions? mixing;
 
-  /// Indicates whether a participant wants to receive mono sound. By default, participants receive stereo audio. This configuration is only applicable when using the Opus codec.
-  bool? preferRecvMono;
-
-  /// Indicates whether a participant wants to send mono sound to a conference. By default, when using the Opus codec, participants' audio is sent as stereo. This configuration is only applicable when using the Opus codec.
-  bool? preferSendMono;
```
Comment (graduad) [inline lib/src/sdk_api/models/conference.dart:217]: Why are we getting rid of these?

File: lib/src/sdk_api/models/conference.dart [hunk 2]
```diff
@@ -534,11 +530,8 @@ class ConferenceListenOptions {
   /// Sets the maximum number of video streams that may be transmitted to the joining participant. The valid parameter values are between 0 and 4 for mobile browsers. By default, the parameter is set to 4.
   num? maxVideoForwarding;
 
-  /// Indicates whether a participant wants to receive mono sound. By default, participants receive stereo audio. This configuration is only applicable when using the Opus codec.
-  bool? preferRecvMono;
-
-  /// Indicates whether a participant wants to send mono sound to a conference. By default, when using the Opus codec, participants' audio is sent as stereo. This configuration is only applicable when using the Opus codec.
-  bool? preferSendMono;
```
Comment (graduad) [inline lib/src/sdk_api/models/conference.dart:541]: Same here. Why deleting these?

Comment (kdrob-dolby) [reply lib/src/sdk_api/models/conference.dart:541]: Those parameters are not being used

File: lib/src/sdk_api/models/conference.dart (hunk 1 above)
Comment (graduad) [reply lib/src/sdk_api/models/conference.dart:217]: The question is why? And also should they be used?

Comment (kdrob-dolby) [reply lib/src/sdk_api/models/conference.dart:217]: React native copied Web SDK parameters and we copied it from react native. There is no equivalent parameters in native mobile SDKs.

**Review by graduad — APPROVED** [review]

**Pull request merged**